conjure-java-core/src/integrationInput/java/com/palantir/product/CookieServiceAsync.java

```diff
@@ -46,4 +47,21 @@ public String toString() {
             }
         };
     }
+
+    /**
+     * Creates an asynchronous/non-blocking client for a CookieService service.
+     */
+    static CookieServiceAsync of(Channel _channel, ConjureRuntime _runtime) {
```

Comment:
I thought about marking this method as deprecated, but it's previously hindered the rollout of conjure-java...